services/movie.py

```diff
@@ -15,13 +16,17 @@ def get_movies(
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        return queryset.filter(title__icontains=title)
```

Comment:
Why do you return here???
```suggestion
        queryset = queryset.filter(title__icontains=title)
```